carla-carma-integration/README.md

```diff
@@ -0,0 +1,95 @@
+# CARAL-CARMA Integration Tool
+This user guide provides step-by-step user instructions on how to build the CARLA-CARMA integration tool on Docker, setup configuration for both CARMA platform and CARLA-CARMA integration tool and run that with CARMA platform.
+
+##  Requirement
+-  Docker (19.03+)
+-  [Nvidia Docker](https://github.com/NVIDIA/nvidia-docker)
+-  [CARMA Platform](https://usdot-carma.atlassian.net/wiki/spaces/CRMPLT/pages/486178827/Development+Environment+Setup) (3.8.0+)
+
+## Setup
+### CARLA-CARMA Integration tool
+1. clone the CARMA simulation repository:
+
+```
+git clone https://github.com/usdot-fhwa-stol/carma-simulation.git
+```
+2. build image from Dockerfile by using following command:
+
+```sh
+cd carma-simulation/carla-carma-integration && ./build.sh
```

Comment:
If we update the script name to build-image.sh for consistency lets update this too.